build.sbt

```diff
@@ -95,8 +107,8 @@ lazy val license = licenses in ThisBuild := Seq(
 lazy val buildSettings = publishing ++
   Seq(
     scalaVersion := "2.12.8",
-    crossScalaVersions := Seq(scalaVersion.value, "2.11.12"),
-    scalacOptions ++= compileFlags,
+    crossScalaVersions := Seq(scalaVersion.value, "2.12.8", "2.11.12"),
```

Comment:
you no longer have cross-compilation for version `2.13.0`, since the main `scalaVersion` is `2.12.8` again :smile:

Reply:
Yeah don't worry I'm doing changes right now

Reply:
It's back now, I need to find a way to make the part using MapView cross compile in rho-swagger (or refactor) and then I think it'll be over.